test/k8s-integration/version.go

```diff
@@ -0,0 +1,113 @@
+package main
```

Comment:
Could https://github.com/kubernetes/apimachinery/blob/master/pkg/util/version/version.go be reused?

Reply:
Thanks for pointing this out